frontend/public/components/routes/utils.ts

```diff
@@ -147,9 +147,7 @@ export const convertEditFormToRoute = (
     },
   };
 
-  if (!_.isEmpty(alternateBackends)) {
-    route.spec.alternateBackends = alternateBackends;
-  }
+  route.spec.alternateBackends = altBackends;
```

Comment:
This fixes the bug as reported in the Jira issue.  Passing an empty array is ok!